src/clarity-angular/datagrid/datagrid-placeholder.spec.ts

```diff
@@ -76,20 +57,11 @@ export default function(): void {
                 expect(context.clarityElement.textContent.trim()).toMatch("Hello world");
             });
 
-            it("has height 0 when no empty rows are needed", function() {
+            it("has height 1px (which acts as a bottom border for the last row)" +
+                "when no empty rows are needed", function() {
                 itemsProvider.all = new Array(2);
                 context.detectChanges();
-                expect(context.clarityElement.scrollHeight).toBe(0);
-            });
-
-            // Yuck. But hey, if we're hard-coding a height, we might as well make sure it we don't mess it up.
-            it("adds 36px to its height for each row needed", function() {
-                pageProvider.size = 10;
-                context.detectChanges();
-                expect(context.clarityElement.scrollHeight).toBe(360);
-                itemsProvider.all = new Array(9);
-                context.detectChanges();
-                expect(context.clarityElement.scrollHeight).toBe(36);
+                expect(context.clarityElement.scrollHeight).toBe(1);
```

Comment:
Good catch but I don't really have a good solution for this right now. I will add a TODO and keep looking for one.